OneSignalSDK/onesignal/src/main/java/com/onesignal/OSFocusHandler.kt

```diff
@@ -87,16 +88,27 @@ class OSFocusHandler {
             .setInitialDelay(delay, TimeUnit.MILLISECONDS)
             .addTag(tag)
             .build()
-        WorkManager.getInstance(context)
-            .enqueueUniqueWork(
-                tag,
-                ExistingWorkPolicy.KEEP,
-                workRequest
-            )
+
+        // Before using WorkManager, check for its existence. Else, initialize it ourselves.
+        if (OSUtils.isWorkManagerInitialized()) {
```

Comment:
Recommend making a helper for this if else logic so it doesn't have to be repeated.

Could be something like `WorkerMangerHelper.getInstance(context)` where it contains this logic.

Reply:
Done